core/src/main/scala/org/http4s/UrlForm.scala

```diff
@@ -106,6 +106,19 @@ object UrlForm {
     x.values.mapValues(_.toList).view.force === y.values.mapValues(_.toList).view.force
   }
 
+  implicit val monoidInstance: Monoid[UrlForm] = new Monoid[UrlForm] {
+    override def empty: UrlForm = UrlForm.empty
+
+    override def combine(x: UrlForm, y: UrlForm): UrlForm =
+      UrlForm(x.values.foldLeft(y.values) {
```

Comment:
I borrowed this implementation from [`MapMonoid`](https://github.com/typelevel/cats/blob/97052b8673afbcd0f8bb79dd7cc273d0e6049bdc/kernel/src/main/scala/cats/kernel/instances/map.scala#L61-L70), more or less inlining the implementation of `Semigroup.maybeCombine`, because [there's no `Monoid[Seq[?]]` instance](https://github.com/typelevel/cats/issues/1222) in cats. Open to alternatives.